apps/src/gamelab/AnimationTab/AnimationTab.jsx

```diff
@@ -72,20 +56,7 @@ const AnimationTab = React.createClass({
             <GameLabVisualizationHeader />
             <AnimationList />
           </div>
-          <div style={styles.framesColumn}>
-            <div className="purple-header workspace-header" style={commonStyles.purpleHeader}>
-              <span>Frames</span>
-            </div>
-            <FrameList />
-          </div>
-          <div style={styles.editorColumn}>
-            <div className="purple-header workspace-header" style={commonStyles.purpleHeader}>
-              <span>Workspace</span>
-            </div>
-            <div style={styles.editorRegion}>
-              TODO: Piskel editor goes here!
-            </div>
-          </div>
+          <iframe id="piskel-frame" style={styles.editorColumn} src="/assets/js/piskel/index.html" />
```

Comment:
Now that I think about it, this path might not work in environments where we fingerprint assets.  I'll revisit this.